cf_xarray/accessor.py

```diff
@@ -365,6 +365,38 @@ def _guess_bounds_dim(da):
     return result
 
 
+def _build_docstring(func):
+    """
+    Builds a nice docstring for wrapped functions, stating what key words
+    can be used for arguments.
+    """
+
+    # this list will need to be updated any time a new mapper is added
+    mapper_docstrings = {
+        _get_axis_coord: f"One or more of {(_AXIS_NAMES + _COORD_NAMES)!r}",
+        _get_axis_coord_single: f"One of {(_AXIS_NAMES + _COORD_NAMES)!r}",
+        _get_measure_variable: f"One of {_CELL_MEASURES!r}",
```

Comment:
@jukent you'll have to add another entry here for the new groupby mapping function